stan/math/rev/meta/conditional_var_value.hpp

```diff
@@ -8,7 +8,9 @@
 namespace stan {
 
 /**
- * Constructs a prim type or var_value from a scalar and a container.
+ * Conditionally construct a var_value container based on a scalar type. For
+ * var types as the scalar, the `var_value<Matrix>`'s inner type will have a
+ *  scalar of a double.
  * @tparam T_scalar Determines the scalar (var/double) of the type.
  * @tparam T_container Determines the container (matrix/vector/matrix_cl ...) of
  * the type. This must be a prim type.
```

Comment:
```suggestion
 * the type.
```
I meant this part. Sorry for the confusion.